metaseq/modules/embedding.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+import torch
+from torch import nn as nn
+
+
+def Embedding(
+    num_embeddings,
+    embedding_dim,
+    padding_idx,
+    initialize_params_on_gpu=False,
+    dtype: Optional[torch.dtype] = None,
+):
```

Comment:
Nit: add a comment here with a high-level description, similar to the encode/decoder classes?